src/public/publicAPIs.ts

```diff
@@ -230,15 +230,27 @@ export function registerAppButtonClickHandler(handler: () => void): void {
 }
 
 /**
- * Registers a handler for hovering the app button.
+ * Registers a handler for entering hover of the app button.
  * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
- * @param handler The handler to invoke when the personal app button is hovered in the app bar.
+ * @param handler The handler to invoke when entering hover of the personal app button in the app bar.
  */
-export function registerAppButtonHoverHandler(handler: () => void): void {
+export function registerAppButtonHoverEnterHandler(handler: () => void): void {
   ensureInitialized();
 
-  GlobalVars.appButtonHoverHandler = handler;
-  handler && sendMessageRequestToParent('registerHandler', ['appButtonHover']);
+  GlobalVars.appButtonHoverEnterHandler = handler;
+  handler && sendMessageRequestToParent('registerHandler', ['appButtonHoverEnter']);
+}
+
+/**
+ * Registers a handler for exiting hover of the app button.
+ * Only one handler can be registered at a time. A subsequent registration replaces an existing registration.
+ * @param handler The handler to invoke when exiting hover of the personal app button in the app bar.
+ */
+export function registerAppButtonHoverLeaveHandler(handler: () => void): void {
+  ensureInitialized();
```

Comment:
Same here; pass `FrameContext.content` to the `ensureInitialized();` call so that developers get notified when their trying to use the API incorrectly.